Curve-finance-suite/New-Fee/README.md

```diff
@@ -0,0 +1,26 @@
+# New Fee Agent
+
+## Description
+
+This agent detects a change in fees to a Curve BasePool or MetaPool by the Curve DAO
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- CURVE-2
```

Comment:
Sorry, I think this wasn't correctly discussed with the team. For the agent number follow the steps from here `https://www.notion.so/nethermind/CurveDAO-7807d9f852ac42f2973c5aa4747a80af#a7033f268df244949d54535ed0f95e29`

Reply:
Addressed in latest commit